backend/src/kitconcept/intranet/locales/de/LC_MESSAGES/kitconcept.intranet.po

```diff
@@ -253,3 +339,8 @@ msgstr "Organisationseinheit"
 #: kitconcept/intranet/profiles/default/registry/querystring.xml
 msgid "label_place"
 msgstr "Ort"
+
+#. Default: "Content Owner:"
+#: kitconcept/intranet/behaviors/lcm.py:27
+msgid "label_responsible"
+msgstr "Inhaltseigentümer:"
```

Comment:
can you give more context, maybe screenshot on how this is used?

Reply:
@jackahl we have form where we are showing it.

<img width="280" height="191" alt="Screenshot 2025-09-01 at 3 59 05 PM" src="https://github.com/user-attachments/assets/bf8acc79-2726-403c-b220-e462f8c578ad" />

Reply:
ok. I think the translation should be ok here then